packages/headless/src/controllers/commerce/product-view/headless-product-view.test.ts

```diff
@@ -0,0 +1,36 @@
+import {ProductRecommendation} from '../../../api/search/search/product-recommendation';
+import {buildMockCommerceState} from '../../../test/mock-commerce-state';
+import {
+  buildMockCommerceEngine,
+  MockedCommerceEngine,
+} from '../../../test/mock-engine-v2';
+import {buildProductView, ProductView} from './headless-product-view';
+
+
+describe('Productview', () => {
+  let engine: MockedCommerceEngine;
+  // TODO: What goes here?
+  let product: ProductRecommendation;
+  let productView: ProductView;
+
+  function initializeProductView() {
+    // TODO: What goes here?
+    product = {};
```

Comment:
The product has only these [three properties](https://github.com/coveo-platform/analytics_schema/blob/master/schemas/common/ec/product.json#L4). We can fill these in when integrating relay,